app/js/controllers/EditCtrl.js

```diff
@@ -969,15 +969,19 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
 					$scope.playerData.frozenProductVersion = $scope.playerData.latestPlayerProductVersion;
 				}
 				else {
-					if($scope.playerData.frozenProductVersion && $scope.playerData.frozenProductVersion != UNKNOWN_PRODUCT_VERSION){
+					if($scope.playerData.frozenProductVersion && $scope.playerData.frozenProductVersion !== UNKNOWN_PRODUCT_VERSION){
 						$scope.playerData.html5Url = JSON.stringify({version: $scope.playerData.frozenProductVersion});
 					}
 				}
 
 				PlayerService.savePlayer($scope.playerData).then(function(value) {
 						localStorageService.remove('tempPlayerID'); // remove temp player from storage (used for deleting unsaved players)
 						apiService.setCache(false);                 // prevent the list controller from using the cache the next time the list loads
-						utilsSvc.alert('Save Player Settings','Player Saved Successfully');
+						if (!$scope.autoUpdate && $scope.playerData.frozenProductVersion !== $scope.playerData.latestPlayerProductVersion){
```

Comment:
also exclude beta

Reply:
There is no need to exclude Beta - It is already excluded in !$scope.autoUpdate